### DISCUSSION TITLE
Power Feeds, outlets sharing a breaker

### DISCUSSION THREAD

**Discussion by bluikko** [netbox-community/netbox#7419] [Q&A] [answered]
How are people treating the "Power Feed" object? Does it represent a breaker or an outlet?

Until now I have treated them as the same since usually there would be one outlet per breaker. But I came across a situation where two outlets share the same breaker. Any ideas how to model this in NetBox? I would like to use the power usage feature in the future so in this case the Power Feed should then represent the breaker and it would need to be somehow split in two.

I guess the only right way is to create a Device, say "ROOM28 outlet B06" that has 2 * Power Outlet and 1 * Power Port?

**Comment by candlerb** [comment, marked as answer]
Yep, that's how I'd do it.

You could just have two power "feeds", but then you'd lose the visibility that they share the same breaker, and therefore are not independent.

**Comment by Sprinterfreak** [comment]
I have following situation:
We have 3P 32A pre-fuse at our supply panel and hard wired PDU devices inside racks with 2 16A breakers per phase in the pdu itself.

At the moment netbox allows to connect multiple feeds to one power port of a PDU, which would be pretty hazardous I guess, but not one feed to multiple power ports.
The ability to wire one feed to multiple power ports would also solve the visibility on the feed.
